### PR TITLE
Make the electrolyzer not a wait game

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
After i collected several feedbacks i saw that there was a choke point onto the path to fusion and metallic hydrogen that was the long wait to build an electrolyzer due to mining lacking personnel, other department taking the gold and silver sheets. It's not fun to have to wait this long just to build a machine to a somewhat basic gas so let's change this.

## Why It's Good For The Game
More fun for atmosians, no longer do the wait game or the steal gold and silver game.

## Changelog
:cl:
add: Added electrolite components
add: Added electrolite components to the node tree
tweak: changed the electrolyzer circuit board to use 2 electrolite, 2 capacitors, 5 cable coils and 1 glass sheet
tweak: reduced h2 gas export gain
imageadd: Added electrolites sprites (Thanks Trollbreeder!)
imageadd: Changed the H2 canister sprite
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
Electrolite part sprites by me.
image

**Comment by Ghilker** [comment]
yup forgot to add in the body of the PR lol just added you in the commit

**Comment by Qustinnus** [comment]
be sure to edit the value of the material if you think itll make it a lot easier to produce.

**Comment by Ghilker** [comment]
the export one?

**Comment by Cornarias** [comment]
Based PR. Thank you for hearing me out. This should make the process infinitely more bearable.

File: code/modules/research/stock_parts.dm [hunk 1]
```diff
@@ -199,6 +205,13 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	rating = 2
 	custom_materials = list(/datum/material/iron=80)
 
+/obj/item/stock_parts/electrolite/adv
+	name = "advanced electrolite"
+	desc = "An electrolite component to separate matters into others"
+	icon_state = "adv_electrolite"
+	rating = 2
+	custom_materials = list(/datum/material/iron = 50, /datum/material/glass = 50)
```
Comment (Dennok) [inline code/modules/research/stock_parts.dm:213]: I think parts adv and highter must take adv materials, like another stock parts

File: code/modules/cargo/exports/large_objects.dm [hunk 2]
```diff
@@ -139,5 +139,5 @@
 	worth += gases[/datum/gas/tritium][MOLES]*5
 	worth += gases[/datum/gas/pluoxium][MOLES]*5
 	worth += gases[/datum/gas/freon][MOLES]*15
-	worth += gases[/datum/gas/hydrogen][MOLES]*25
+	worth += gases[/datum/gas/hydrogen][MOLES]*10
```
Comment (Dennok) [inline code/modules/cargo/exports/large_objects.dm:142]: Silent hydrogen debuf?

Comment (Ghilker) [reply code/modules/cargo/exports/large_objects.dm:142]: uh no? i wrote that in the changelog too

File: code/modules/research/stock_parts.dm (hunk 1 above)
Comment (Ghilker) [reply code/modules/research/stock_parts.dm:213]: wym?

File: code/modules/cargo/exports/large_objects.dm (hunk 2 above)
Comment (Dennok) [reply code/modules/cargo/exports/large_objects.dm:142]: Ups, i lost it.

File: code/modules/research/stock_parts.dm (hunk 1 above)
Comment (Dennok) [reply code/modules/research/stock_parts.dm:213]: I misunderstand something here.

**Pull request merged**